### DISCUSSION TITLE
Blank article about the SplitView Component

### DISCUSSION THREAD

**Discussion by Kuprich** [AvaloniaUI/Avalonia#13940] [General]
Hi, guys!
I was looking for information on the *SplitView* component
But this [article](https://docs.avaloniaui.net/ru/docs/guides/development-guides/how-to-show-and-hide-a-split-view-pane-with-mvvm) was empty.

As a result, I created [this demo](https://github.com/Kuprich/AvaloniaSplitViewDemo ) example myself.
Maybe this simple example will help someone

Thank you for creating a cool project named **"Avalonia"**

**Comment by maxkatz6** [comment]
Please open an issue in avalonia-docs repository
